presto-raptor/src/main/java/com/facebook/presto/raptor/RaptorMetadata.java

```diff
@@ -424,7 +424,14 @@ public void createTable(ConnectorSession session, ConnectorTableMetadata tableMe
     @Override
     public void createTable(ConnectorSession session, ConnectorTableMetadata tableMetadata, boolean failIfExists)
     {
-        createTable(session, tableMetadata);
+        try {
```

Comment:
We shouldn't need to do anything for Raptor or other non-transactional connectors, since the table creation is done directly in the `createTable()` call and thus the normal handling in the engine will cover it.